analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.CSharp8_NullableEnable.cs

```diff
@@ -0,0 +1,81 @@
+﻿#nullable enable
```

Comment:
Let's put this into the existing C# files. There's quite a lot of files already for this rule

Reply:
This would add a lot of warnings to the file and makes test isolation more difficult.
We need different files here in the long run anyway, because we need to run against .Net core (annotated) and .Net framework (not annotated) or .NetStandard 2.0 reference assemblies (not annotated) and .NetStandard 2.1 (annotated) e.g. when we implement support for `[NotNullIfNotNull]` and the like where we want to explicitly turn on or off nullable annotations support. We will end up with tree modes:

| #nullable   | Annotated BCL  | Not annotated BCL |
|---|---|---|
| disable | a | a |
| enable | b | c |

Reply:
Let's use `.Nullable.cs` suffix then (and rename the UT accordingly)